github/repos_rules.go

```diff
@@ -936,32 +965,55 @@ func (s *RepositoriesService) UpdateRuleset(ctx context.Context, owner, repo str
 	return ruleset, resp, nil
 }
 
+// UpdateRulesetClearBypassActor clears the ruleset bypass actors for a ruleset for the specified repository.
+//
+// This function is necessary as the UpdateRuleset function does not marshal ByPassActor if passed as nil or an empty array.
+//
+// GitHub API docs: https://docs.github.com/rest/repos/rules#update-a-repository-ruleset
+//
+//meta:operation PUT /repos/{owner}/{repo}/rulesets/{ruleset_id}
+func (s *RepositoriesService) UpdateRulesetClearBypassActor(ctx context.Context, owner, repo string, rulesetID int64) (*Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/rulesets/%v", owner, repo, rulesetID)
+
+	rsClearBypassActor := rulesetClearBypassActors{}
+
+	req, err := s.client.NewRequest("PUT", u, rsClearBypassActor)
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := s.client.Do(ctx, req, nil)
+	if err != nil {
+		return resp, err
+	}
+
+	return resp, nil
+}
+
 // UpdateRulesetNoBypassActor updates a ruleset for the specified repository.
 //
 // This function is necessary as the UpdateRuleset function does not marshal ByPassActor if passed as nil or an empty array.
```

Comment:
I think this comment needs updating since it is no longer possible to pass `nil` with this PR change.

In fact, with this deprecation notice, it is necessary to modify this endpoint at all?

Reply:
Ha ha ha, I undid the changes. Do you want me to put them back?

Reply:
SORRY!!!  You are _FAST_!!!

Your call.  I'm fine either way.

Reply:
I've updated for consistency and updated the comment as it refers to the default update method. I've also renamed the public facing `rs` to `ruleset`.